Source/Makefile

```diff
@@ -0,0 +1,123 @@
+## Haiku Generic Makefile ##
+
+## Fill in this file to specify the project being created, and the referenced
+## Makefile-Engine will do all of the hard work for you. This handles any
+## architecture of Haiku.
+
+# The name of the binary.
+NAME = 2dPhysicsDemo
+
+# The type of binary, must be one of:
+#	APP:	Application
+#	SHARED:	Shared library or add-on
+#	STATIC:	Static library archive
+#	DRIVER: Kernel driver
+TYPE = APP
+
+# 	If you plan to use localization, specify the application's MIME signature.
+APP_MIME_SIG = 
```

Comment:
Please always add the signature in Makefiles: application/x-vnd.2dPhysicsDemo

Reply:
Oops! I forgot this time. :(